src/main/java/software/amazon/awssdk/crt/http/Http2ClientConnection.java

```diff
@@ -26,19 +30,9 @@
      * (RFC-7540 7).
      */
     public enum Http2ErrorCode {
-        PROTOCOL_ERROR(1),
-        INTERNAL_ERROR(2),
-        FLOW_CONTROL_ERROR(3),
-        SETTINGS_TIMEOUT(4),
-        STREAM_CLOSED(5),
-        FRAME_SIZE_ERROR(6),
-        REFUSED_STREAM(7),
-        CANCEL(8),
-        COMPRESSION_ERROR(9),
-        CONNECT_ERROR(10),
-        ENHANCE_YOUR_CALM(11),
-        INADEQUATE_SECURITY(12),
-        HTTP_1_1_REQUIRED(13);
+        PROTOCOL_ERROR(1), INTERNAL_ERROR(2), FLOW_CONTROL_ERROR(3), SETTINGS_TIMEOUT(4), STREAM_CLOSED(5),
```

Comment:
How are these errors surfaced back to the caller?

Curious about things where the peer might send an error in the middle of a request like in a `GOAWAY`

Reply:
>@TODO: bindings for getters of local/remote setting and goaway.
https://github.com/awslabs/aws-crt-java/pull/398/files#diff-f99e160a2ce01eefcfa64cf86b4611d7eae8cf2851f37627682b7334f2e9a269R154
If it's something we want, I'll add the binding soon.

Reply:
We also have `on_goaway_received` callback in C, but it will be bound after the connection pool http2 support.